types/three/src/objects/Skeleton.d.ts

```diff
@@ -112,4 +112,8 @@ export class Skeleton {
      * Call this method whenever this instance is no longer used in your app.
      */
     dispose(): void;
+    
+    toJSON(): any;
+    
+    fromJSON(json: any, bones: Record<string, Bone>): void;
```

Comment:
Why? I think i'd be best to do type return for toJSON once we do type return for every other classes that have toJSON

like 
Object3D.toJSON
BufferGeometry.toJSON

Reply:
To avoid the use of `any`, since it can cause surprising results if a user doesn't realize it's `any`. If we don't want to type it fully, then let's just use `unknown` instead of `any` to avoid the appearance of being typed.

Reply:
As for whether to wait until we do it for all the other classes, I'm more inclined to think doing the right thing here is more important than consistency across classes. If we wait until we can do it for all the classes at the same time, it's less likely to happen in a timely manner in my experience. Better to just get it right where we can and fix the other classes later.

Reply:
> As for whether to wait until we do it for all the other classes

![image](https://user-images.githubusercontent.com/45743294/234404954-ddcd3f67-0279-4189-8726-f2f3a0c8ce86.png)

![image](https://user-images.githubusercontent.com/45743294/234404976-b6cf9f19-55bb-410e-9bb4-b8ef4fef4ef7.png)

😏 😏 😏 😏 😏

Reply:
If i where to push this, how would i go about it?? have a dedicated ts files for all of these jsons types and use it something like this
`Sterilized.Object3D`

Or have each in their corresponding files like

export interface `Object3DJSON`

idk your choice

Reply:
Ha, nice work! My preference is to keep them in the corresponding files with a name like `Object3DJSON` to keep it as close to the source as possible.

Reply:
> Ha, nice work! My preference is to keep them in the corresponding files with a name like `Object3DJSON` to keep it as close to the source as possible.

Alr, i think you should accept this pull request as is, then i will add the JSON types for everything including Skeleton in a new pull request, would make things easier.

I'll change it to unkown for now instead of any like you said

Reply:
Thanks.